### ISSUE TITLE
Re-ensure IME functionality on top of QML frontend port

### ISSUE THREAD

**Issue by christianparpart** [contour-terminal/contour#1136]
Since IME seems to be inherently different when porting from QtWidgets to QtQuick / QML, let's tackle this in its own ticket after #761 was merged but required for a next major release.

I need to remember how to make use of IME locally again (it required some additional client side setup).

## Action items

- [ ] ensure IME is working on top of QML frontend port
- [ ] Add documentation on how to make use of IME in the user docs (fun fact, also helpful for devs 😄)
- [x] ~~I sadly don't have any idea how this is testable~~
